functionary/core/management/commands/utils.py

```diff
@@ -0,0 +1,29 @@
+from time import sleep
+from typing import Callable
+
+from django.db.utils import ProgrammingError
+
+SLEEP_DURATION = 2
```

Comment:
Let's go with 5.  I don't know why, but 2s feels like it's below my "too spammy" threshold for this.

Reply:
Fixed